src/config/config.yml

```diff
@@ -68,3 +68,5 @@ mot:
     rwt: RWT.pdf
     adr_pass: ADR_PASS.pdf;
   documentDir: CVS
+welsh:
+  secret_key: feature/cert-gen-smc/values
```

Comment:
Just to confirm, this being used specifically for local testing / running and when running on feature environments / develop we don't want to have a remote section like how s3 currently does it

Reply:
Confirmed, most testing will be done on lambda